src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "CARTS")
+public class Cart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @NotNull
+    @Column(name = "ID",unique = true)
+    private Long id;
+
+    @Column(name = "NAME")
+    private String name;
```

Comment:
Co będzie oznaczać `name`? Brakuje "powiązania" z produktem/produktami. Nie mam na myśli bezpośredniej relacji. Może `name` to `name` produktu?